packages/inferno-compat/src/index.js

```diff
@@ -16,11 +16,15 @@ const ARR = [];
 
 const Children = {
 	map(children, fn, ctx) {
+		if (children === undefined) { return undefined; }
+		if (children === null) { return null; }
```

Comment:
this could be simplified to isNullOrUndef(children) {return children} it also minified better and V8 will inline function call anyway